packages/connect/src/types/settings.ts

```diff
@@ -34,6 +34,8 @@ export interface ConnectSettingsPublic {
     deeplinkOpen?: (url: string) => void;
     deeplinkCallbackUrl?: string;
     deeplinkUrl?: string;
+    // URL for binary files such as firmware, may be local or remote. If not provided, firmware hash check won't be done.
+    binFilesBaseUrl?: string;
```

Comment:
this is basically `baseUrl` which is part of some calls right? do these files need to be bin?

Reply:
yes. But because the fw hash check is to be done before discovery, I placed it in `getFeatures()`, and so I need to supply it as `connect` setting.
This way it _also_ fulfills another purpose - if **not** provided, fw hash check will be skipped, which, AFAIK, is essential for 3rd party users of `connect`. We want it only for suite desktop & web.
- on desktop (dev) it points to `./packages/suite-desktop/build/static/bin/`
- on web it points to `/static/connect/data`

Reply:
it makes sense. Maybe it should be used also in other calls? at least those that accept `baseUrl` param? And if yes, what should take precedence?

Reply:
It'd make sense to me that those connect calls which currently receive `baseUrl` param, should use the settings baseUrl by default, and have the param optional which would override - this approach would not break API.

I added it to TODOs in PR description.